pulp_2to3_migration/app/serializers.py

```diff
@@ -169,7 +169,7 @@ def to_representation(self, instance):
         """
         result = super(Pulp2ContentSerializer, self).to_representation(instance)
         if not result.get('pulp3_repository_version'):
-            del result['pulp3_repository_version']
+            result.pop('pulp3_repository_version', None)
```

Comment:
Can we extend the doc string to explain the why, plz.
E.g. repo version is set only for content which can migrate from one pulp 2 content unit into multiple pulp 3 content units.
Serializing it when it is not set can mislead users that content doesn't belong to any repo_version.

Reply:
+1